third-party/chpl-venv/chplcheck-requirements.txt

```diff
@@ -4,3 +4,5 @@ lsprotocol==2023.0.1
 pygls==1.3.0
 typeguard==3.0.2
 ConfigArgParse==1.7
+pytest==8.2.0
+pytest-lsp==0.4.1
```

Comment:
This makes me pause a little. Why are we now requiring all users of `cls`/`chplcheck` to have dependencies that are only used for development testing purposes. 

In my opinion, these should be in a separate venv (like `test-venv`). Maybe `chapel-py-test-venv`?